charts/newrelic-infrastructure/values.yaml

```diff
@@ -412,22 +376,12 @@ podLabels: {}
 
 # -- Pod scheduling priority
 # Ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
-priorityClassName: ""
-
-# -- Security context used in all the containers of the pods
-# When `privileged == true`, the Kubelet scraper will run as root and ignore these settings.
-# @default -- See `values.yaml`
-securityContext:
-  # OpenShift users might want to change the default UID:GID to be inside the allowed range.
-  runAsUser: 1000
-  runAsGroup: 2000
-  allowPrivilegeEscalation: false
-  readOnlyRootFilesystem: true
```

Comment:
`securityContext` is probably the only default worth hiding behind a template, as we want it to change depending on `privileged`.

Regardless, I think we should leave the blank entry:

```yaml
// Very nice coment explaining why these are blank and how defaults change.
podSecurityContext:
containerSecurityContext:
```

Reply:
I leave them for the last nit-picking commit.

Reply:
Solved in [#881654c](https://github.com/newrelic/nri-kubernetes/pull/383/commits/881654c7b9e3b6a104b4a9a29652cf7526e4c4f8)